client/src/pages/MemberProfilePage/MemberProfilePage.css

```diff
@@ -22,10 +22,9 @@
   text-align: left;
 }
 
-.MemberProfilePage .Groups__list {
+.MemberProfilePage .Groups {
```

Comment:
This is still overwriting the styling of another component. Anywhere you're doing this, it means you need to go and update that component, because you've found a case that it can't handle